decidim-sortitions/app/commands/decidim/sortitions/admin/create_sortition.rb

```diff
@@ -58,6 +58,8 @@ def categorize(sortition)
             decidim_category_id: form.decidim_category_id,
             categorizable: sortition
           )
+          # reload sortition model to make it aware of the category
+          sortition.reload_category
```

Comment:
If we create the categorization via the sortition I think Rails will take care of it:

```ruby
sorition.categorizations.create(category: Decidim::Category.find(form.decidim_category_id))
```

PS: I think the form should have a `category` method that returns the instance instead of passing ids around.

Reply:
Maybe this could be done inside the `create_sortition` method as another attribute.

Reply:
just passing the category as an attribute works.